packages/terra-functional-testing/src/commands/viewport-helpers.js

```diff
@@ -46,23 +46,19 @@ const describeViewports = (title, viewports, fn) => {
   let localViewports = viewports;
 
   // If formFactor is defined in the service options/env and viewports contains this form factor, run that size else don't run the tests
-  const [, options = {}] = global.browser.options.services.find(([service]) => (
-    typeof service === 'function' && service.name === 'TerraService'
-  ));
-
-  const { formFactor } = options;
+  const { formFactor } = global.Terra.serviceOptions;
   if (formFactor) {
     localViewports = viewports.includes(formFactor) ? [formFactor] : [];
   }
 
   let currentViewportSize;
   localViewports.forEach(viewport => global.describe(`[${viewport}]`, () => {
-    global.beforeAll(() => {
+    global.before(() => {
```

Comment:
:/ bummer

Reply:
This is using mocha. `beforeAll` and `afterAll` are not their APIs so switching back to `before` and `after`.